platforms/common/include/px4_platform_common/defines.h

```diff
@@ -99,7 +99,10 @@ __END_DECLS
 
 #define PX4_ROOTFSDIR CONFIG_BOARD_ROOTFSDIR
 
+// Qurt doesn't have an SD card for storage
+#ifndef __PX4_QURT
 #define PX4_STORAGEDIR PX4_ROOTFSDIR
```

Comment:
This file is an example of something I think should be entirely split per platform.
Then QuRT simply wouldn't have PX4_STORAGEDIR or PX4_ROOTFSDIR in the first place, rather than having to opt out of it.